CMSgov/onemac
# Oy2 7281 header footer

Story: https://qmacbis.atlassian.net/browse/OY2-7281

Endpoint URL: https://d3vpd9enuir5py.cloudfront.net

Changes:
Created the Home page header and footer components as appears in the Figma link.
Renamed "About" link text to "Home"
Made sure the Dashboard link appears only if the user is logged-in
The selected tab/link is underlined as displayed in the Figma file and the other links are not underlined.

*** Note: Since the Home page is not yet updated, the page with the new header and footer will look not so pleasing at this time.

Testing:
- Load the application in the browser and verify the Home page header and footer appear as in the Figma design - -(New Homepage Design - Figma file: https://www.figma.com/file/OOtw6NirwsWqUlYkVW9pdz/Launch?node-id=252%3A7).
- Verify if the 'About' Link text is replaced 'Home'.
- Make sure the Dashboard link appears only if the user is logged-in
